lighthouse-core/test/test-utils.js

```diff
@@ -138,9 +138,26 @@ function makeParamsOptional(fn) {
   return /** @type {(...args: RecursivePartial<TParams>) => TReturn} */ (fn);
 }
 
+/**
+ * Mocks gatherers for BaseArtifacts that tests for components using GatherRunner
+ * shouldn't concern themselves about.
+ */
+function makeMocksForGatherRunner() {
```

Comment:
Tried to share this code but seems `jest.mock` doesn't work the same when called from a different module. Unless someone has an idea, I'll give up on this.

Reply:
#11664 explains why